Dockerfile

```diff
@@ -1,62 +1,49 @@
 # To run: docker run -v /path/to/wsgi.py:/var/www/pidgin/wsgi.py --name=pidgin -p 81:80 pidgin
 # To check running container: docker exec -it pidgin /bin/bash
 
-FROM ubuntu:16.04
 
-ENV DEBIAN_FRONTEND=noninteractive
+FROM tiangolo/uwsgi-nginx:python3.6-alpine3.7
+
```

Comment:
Why don't we just copy this guy's Dockerfile to `~/cloud-automation/Docker/uwsgi-nginx`, and make our own `alpine` base image that all of our python services inherit from?